domain/src/main/java/com/github/leuludyha/domain/util/Result.kt

```diff
@@ -0,0 +1,12 @@
+package com.github.leuludyha.domain.util
+
+import java.io.Serializable
+
+sealed class Result<T>(
```

Comment:
It seems to me that  the Result data class is part of domain model, why create a new util package ?